### ISSUE TITLE
Nothing clickable to link to front-page headings

### ISSUE THREAD

**Issue by hatched** [canonical-web-and-design/multipass.run#80]
The `h2` markup doesn't provide an id that can be used to link directly to sub headings like "Install" and "get started"

**Comment by Saviq** [comment]
+1, @ricab this one's for you :)

**Comment by Saviq** [comment]
For reference: https://forum.snapcraft.io/t/adding-named-anchors-in-docs/13853

**Comment by ricab** [comment]
Yup +1, this would indeed be nice :)

**Comment by matthewpaulthomas** [comment]
As far as I can tell, the only page on multipass.run that has “Install” and “Get started” headings is the front page. There, the `h2`s don’t have `id`s, but the `section`s do:
https://multipass.run/#install
https://multipass.run/#get-started
https://multipass.run/#learn-more

Saviq’s linked thread discusses “docs”, where @degville pointed out that it’s not specific to Multipass. So if this issue is really about Discourse-generated docs in general, I suggest reporting it in [canonicalwebteam.discourse-docs](https://github.com/canonical-web-and-design/canonicalwebteam.discourse-docs).

**Comment by hatched** [comment]
Yes this was for the main page, not the docs. It's not intuitive to have to link to a section especially because there are no anchors to generate the links.

**Comment by matthewpaulthomas** [comment]
Well, ok, that deserves more thought. Other sites sometimes have visible anchor links at the end of headings. However:
- That’s usually in long reference documents, not on a site’s front page.
- If we did it on multipass.run, probably we should do it on microstack.run and microk8s.io too. (Not simultaneously, but at least planned.)

**Comment by evilnick** [comment]
It would also be nice on ubuntu.com/kubernetes/docs (not discourse generated)

**Comment by hatched** [comment]
A naive fix for this might be simply to make the headings an anchor tag to respective sections...`#install` for example.

**Comment by matthewpaulthomas** [comment]
If you mean “give them an `id=` attribute”, the sections already have those, and anyone looking at the HTML will find them.

If you mean “make them `<a href` links”, that would make them blue, and underlined on hover, which would be rather off-brand. I think we’d need a more subtle way of advertising that they’re anchors.

**Comment by hatched** [comment]
The blue & underline is strictly a design decision, it could be anything even have no hover or colour change effect - only to enable the deep linking.

**Comment by g1964j** [comment]
On prend répit c est tout avant pour tout le monde.

**Comment by carkod** [comment]
Conversation seems to be stale, reopen if needed

**Comment by carkod** [comment]
@davegoddard42 would you be able to take a look at this see what's the best approach?

**Comment by davegoddard42** [comment]
@carkod I'll do some investigation, and raise it in the Vanilla Working Group.

**Comment by anthonydillon** [comment]
Heads up, this has been requested and validated by the working group.
https://github.com/canonical-web-and-design/vanilla-framework/issues/3863

**Comment by davegoddard42** [comment]
Great - cheers @anthonydillon.
